examples/xml-subtyping/xml_subtyping.description

```diff
@@ -1,10 +1,12 @@
-// An `xml` value belongs to `xml:Element` if it consists of just an element
-// item. Similarly for `xml:Comment` and `xml:ProcessingInstruction`.
-// An `xml` value belongs to `xml:Text` if it consists of a text item or is 
-// empty.
-// An `xml` value belongs to the type `xml<T>` if each of its members belong 
-// to `T`.
-// Functions in lang.xml use this to provide safe and convenient typing.
-// For example, `x.elements()` returns element items in `x` as type 
-// `xml<xml:Element>` and `e.getName()` and `e.setName()` are defined when 
-// `e` has type `xml:Element`.
+// <ul>
+// <li>An `xml` value belongs to the `xml:Element` if it consists of just one element item.</li>
+// <li>It is the same for the `xml:Comment` and `xml:ProcessingInstruction`.</li>
+// <li>An `xml` value belongs to `xml:Text` if it consists of a text item or is empty.</li>
+// <li>An `xml` value belongs to the type `xml<T>` if each of its members belong to `T`.</li>
+// <li>Functions in the `lang.xml` use this to provide safe and convenient typing. For example,
+// <ul>
+// <li>`x.elements()` returns element items in `x` as type `xml<xml:Element>.`</li>
+// <li>`e.getName()` and `e.setName()` are defined when `e` has the type `xml:Element`.</li>
+// </ul>
+// </li>
+// </ul>
```

Comment:
Er why change these paras to lists? 🤔

Reply:
Having them as point form is easy to read. That's why I did this change. If not let's ignore this change

Reply:
I added a comment for https://github.com/ballerina-platform/ballerina-distribution/pull/2550#discussion_r795043364 also. IMO, the entire description being a list may not look nice.

Anyway these are repeated in-line also right? Maybe eventually we can replace the current text with a better overview, but I don't think we should be making the whole thing a list now.

Reply:
Let's close this. It is good to keep it in the existing way